backend/src/plugins/ModActions/commands/CasesModCmd.ts

```diff
@@ -50,17 +49,18 @@ export const CasesModCmd = modActionsCmd({
         const cases = await casesPlugin.getRecentCasesByMod(modId, casesPerPage, (page - 1) * casesPerPage);
         const lines = await asyncMap(cases, (c) => casesPlugin.getCaseSummary(c, true, msg.author.id));
 
+        const isLastPage = page === totalPages;
         const firstCaseNum = (page - 1) * casesPerPage + 1;
-        const lastCaseNum = page * casesPerPage;
+        const lastCaseNum = isLastPage ? totalCases : page * casesPerPage;
         const title = `Most recent cases ${firstCaseNum}-${lastCaseNum} of ${totalCases} by ${modName}`;
 
         const embed = {
           author: {
             name: title,
             icon_url: mod instanceof User ? mod.displayAvatarURL() : undefined,
           },
+          description: lines.join("\n"),
```

Comment:
We should make sure this is below 4096 characters

Reply:
That would require some major refactoring of the createPaginatedMessage since it's stateless and we just get a fixed # of cases per page currently, would have to offset the next page by X without being able to save this state anywhere.
Probably just lower cases per page, or implement ZDEV-62